applications/nginx-search/src/main/docker/nginx.conf

```diff
@@ -102,6 +102,11 @@ http {
             proxy_pass http://api-cat:8080;
         }
 
+        location /concept-cat {
```

Comment:
Why not use /concepts here. It is the logical path!

Reply:
god idé! takk!